src/hashmap.c

```diff
@@ -137,7 +137,7 @@
     kh_clear(T##_##U##_map, map->table);                                      \
   }
 
-MAP_IMPL(string, ptr_t, DEFAULT_INITIALIZER)
+MAP_IMPL(string, ptr_t, DEFAULT_INITIALIZER) /* maps pluginkey <> connection */
 MAP_IMPL(cstr_t, ptr_t, DEFAULT_INITIALIZER)
 MAP_IMPL(string, dispatch_info, {.func = NULL, .async = false, .name = {.str = NULL, .length = 0}})
-MAP_IMPL(uint64_t, string, {.str = NULL, .length = 0})
+MAP_IMPL(uint64_t, string, DEFAULT_INITIALIZER) /* maps callid <> pluginkey */
```

Comment:
looks wrong to me. why not use a default initializer for the `string` object?:

`{.str = NULL, .length = 0}`